<?xml version="1.0"?>
<launch>
    <node pkg="sq_scan_matching" type="icp_sq_scan_matcher" name="icp_sq_scan_matcher" output="screen">
        <param name="correspondance_distance" value="0.2"/>
        <param name="matching_score" value="0.5"/>
        <param name="max_iterations" value="50"/>
    </node>
    <node pkg="sq_scan_matching" type="ekf" name="ekf" output="screen">
        <param name="init_x" value="0.0"/>
        <param name="init_y" value="0.0"/>
        <param name="init_z" value="0.0"/>
        <param name="init_roll" value="0.0"/>
        <param name="init_pitch" value="0.0"/>
        <param name="init_yaw" value="0.0"/>
        <param name="init_sigma" value="0.0001"/>
        <param name="sigma_imu" value="0.0001"/>
        <param name="sigma_odom" value="0.0001"/>
        <param name="sigma_match" value="0.0001"/>
    </node>
</launch>
